src/waveresponse/_core.py

```diff
@@ -1460,28 +1460,38 @@ def extreme(self, t, q=0.37):
         of the process, and ``q`` is the quantile. Setting ``q=0.37`` yields the
         most probable maximum (MPM).
 
-        The extreme value indicates a level which the maximum value of the process
-        amplitudes will be below with a certain probability. Note that the method
-        only computes extreme values for the maxima, and does not consider the minima.
-
         Parameters
         ----------
         t : float
             Time/duration in seconds for which the of the process is observed.
         q : float or array-like
             Quantile or sequence of quantiles to compute. Must be between 0 and 1
             (inclusive).
+        absmax : bool
+            Whether to compute absolute value extremes (i.e., taking the minima into account).
+            If ``False`` (default), only the maxima are considered. See Notes.
 
         Returns
         -------
         x : float or array
-            Extreme value(s). During the given time period, the maximum value of
-            the process amplitudes will be below the returned value with a given
-            probability.
+            Extreme value(s). During the given time period, the maximum value (or
+            absolute value maximum) of the process amplitudes will be below the
+            returned value with the given probability.
+
+        Notes
+        -----
+        Computing absolute value extremes by setting ``absmax=True`` is equivalent
+        to doubling the zero-crossing frequency, ``fz = 1 / Tz``.
```

Comment:
```suggestion
        to doubling the expected zero-crossing rate, ``fz = 1 / Tz``.
```

zero-crossing rate is more common in the litterature